packages/kyt-core/cli/__tests__/index.test.js

```diff
@@ -5,7 +5,8 @@ jest.mock('../commands', () => {});
 it('bails when min node version is not met', () => {
   global.console.error = jest.fn();
   global.console.info = jest.fn();
-  global.process = { versions: { node: '4.0' } };
+  delete global.process.versions.node;
```

Comment:
can't change the value otherwise - I am assuming there is some `Object.defineProperty` behavior deep in the bowels of Node that is restricting direct mutability of these values

Reply:
ok!